### PR TITLE
Link to command_line sample

### PR DESCRIPTION
**Pull request by kkreine**
This adds a link to the command_line sample in the command line tutorial doc. I'm happy to make any requested changes.

Fixes one part of #3221

### CODE REVIEW

**Comment by kkreine** [comment]
I apologize; I thought I had signed the CLA but did not prior to submitting this PR. I have signed it now.

File: src/_tutorials/server/cmdline.md [hunk 1]
```diff
@@ -572,7 +572,9 @@ For more classes, functions, and properties,
 consult to the API reference for
 [dart:io,]({{ioAPI}}/dart-io-library.html)
 [dart:convert,]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-convert/dart-convert-library.html)
-and the [args]({{argsAPI}}/args-library.html) package.
+and the [args]({{argsAPI}}/args-library.html) package.  
+
+For an example of a command line app, see the [command_line](https://github.com/dart-lang/samples/tree/master/command_line) sample.
```
Comment (parlough) [inline src/_tutorials/server/cmdline.md:577]: One small suggestion is to modify the addition to use [semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks). We use them to make reading and editing in source more clear. I also split the link declaration in the suggestion to make the line more readable.

```suggestion
For another example of a command line app, 
see the [command_line][] sample.

[command_line]: https://github.com/dart-lang/samples/tree/master/command_line
```

**Review by parlough — APPROVED** [review]
Thanks so much @kkreine! One suggestion around formatting, but overall looks great 👍

File: src/_tutorials/server/cmdline.md (hunk 1 above)
Comment (parlough) [reply src/_tutorials/server/cmdline.md]: Thanks for making the updates!

Comment (kkreine) [reply src/_tutorials/server/cmdline.md]: Thanks for the feedback @parlough! I made these changes. Let me know if there's anything further to do.

**Comment by parlough** [comment]
This looks great, thanks so much @kkreine! 

I'll merge this despite the freeze as it's a relatively simple Markdown/link addition (if anyone needs me to revert this, let me know).

**Pull request merged**